core/src/voting_service.rs

```diff
@@ -86,7 +90,14 @@ impl VotingService {
         } else {
             crate::banking_stage::next_leader_tpu(cluster_info, poh_recorder)
         };
-        let _ = cluster_info.send_transaction(vote_op.tx(), target_address);
+
+        if use_quic_client {
```

Comment:
let's time this since we're switch to a stateful protocol

Reply:
done